### PR TITLE
🗃️(request-for-access): added requestAccess database and the data access

### PR DESCRIPTION
**Pull request by phadimanelerefolo**

### CODE REVIEW

**Comment by LiamBurgess18015001** [comment]
What changed in yarn.lock?

The whole system can break in yarn.lock changed

**Comment by arulagrawal** [comment]
Your branch is extremely outdated. Please update your branch by removing your yarn.lock, then re-add whatever you need.

File: libs/api/request-access/repository/feature/src/lib/RequestAccessRepository.spec.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          ItemId: 'ACADEMIC_RECORD',
+          Accepted: false
+          }});
+    });
+    it('generic test', async () => {
+      const call1 = await data.insertRequest("1322","1322","CV",false);
+      const call2 = await data.findRequestByStudIdCompId("1322","1322");
+      console.log(call1);
+      console.log(call2[0]);
+      console.log(call2[1]);
```
Comment (arulagrawal) [inline libs/api/request-access/repository/feature/src/lib/RequestAccessRepository.spec.ts:27]: Unnecessary logs

File: libs/api/request-access/repository/feature/src/lib/RequestAccessRepository.spec.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+     expect(true).toBeTruthy();
```
Comment (arulagrawal) [inline libs/api/request-access/repository/feature/src/lib/RequestAccessRepository.spec.ts:28]: What is this? Shouldn't we be testing something related to our code?

**Review by arulagrawal — CHANGES_REQUESTED** [review]

File: libs/api/request-access/repository/feature/src/index.ts [hunk 3]
```diff
@@ -1 +1 @@
-export * from './lib/api-request-accesss-repository-feature.module';
+export * from './lib/api-request-access-repository-feature.module';
```
Comment (arulagrawal) [inline libs/api/request-access/repository/feature/src/index.ts:1]: This file being referenced is essentially an empty module. Is this intended?

**Review by arulagrawal — CHANGES_REQUESTED** [review]

File: libs/api/request-access/repository/feature/src/index.ts (hunk 3 above)
Comment (arulagrawal) [reply libs/api/request-access/repository/feature/src/index.ts:1]: Shouldn't the module be set so the repository is a provider and is exported?

**Review by arulagrawal — APPROVED** [review]
I guess the actual test can be another PR.

**Pull request merged**